### DISCUSSION TITLE
allsky_dewheater.py for the RPi5 with gpiod instead of GPIO

### DISCUSSION THREAD

**Discussion by Adler6907** [AllskyTeam/allsky-modules#36] [Show and tell]
I modified `allsky_dewheater.py` slightly so that the heating relay can be switched via a GPIO again.
 
I replaced import `RPi.GPIO as GPIO` with import `gpiod`. `gpiod` is already preinstalled with bookworm.
Changes were made in `allsky_dewheater.py` in the functions `turnHeaterOff`, `turnHeaterOn` and `dewheater`. The `setmode` function was no longer necessary.
In `allsky_shared.py` the function `getGPIOPin` is no longer necessary for `allsky_dewheater.py`.
In the attachment `allsky_dewheater.py` which only needs to be replaced in `/opt/allsky/modules
[allsky_dewheater.py.txt](https://github.com/AllskyTeam/allsky-modules/files/13694157/allsky_dewheater.py.txt)
`

**Comment by Alex-developer** [comment]
Thanks for that, 

Have a look at the dev branch as this has all of the fixes in it for Bookworm and the pi 5, we are aiming to release this in the next week
